overlays/DIMO/aftermarket.vspec

```diff
@@ -0,0 +1,96 @@
+# cell provides fields related to dimo device's cellular signal.
+# Dependencies to other overlays: None
+# Known conflicts with other overlays: None
+#
+HDOP:
+  datatype: float
+  type: sensor
+  description: Horizontal dilution of precision of GPS
+
+NSAT:
+  datatype: float
+  type: sensor
+  description: Number of sync satellites for GPS
+
+WPAState:
+  datatype: string
+  type: sensor
+  description: Indicate the current WPA state for the device's wifi
+
+SSID:
+  datatype: string
+  type: sensor
+  description: Service Set Identifier for the wifi.
+
+GSMUMTSSignalLevel:
+  type: sensor
+  datatype: float
+  description: Device network signal level
+
+Power:
+  type: branch
+  description: device power status
+
+Power.BatteryVoltage:
+  type: sensor
+  datatype: float
+  description: Device onboard battery voltage
+
+ECO:
+  type: branch
+  description: ruptela events aggregated
+
+ECO.BrakeCounter:
+  type: sensor
+  datatype: float
+  description: Brake event counter
+
+ECO.HarshBrakeCounter:
+  type: sensor
+  datatype: float
+  description: Harsh brake event counter
+
+ECO.HarshAccelerationCounter:
+  type: sensor
+  datatype: float
+  description: Harsh acceleration event counter
+
+ECO.IdlingTimer:
+  type: sensor
+  datatype: float
+  description: Length of time vehicle engine was idling since last start up
+
+ECO.IdlingEvent:
+  type: sensor
+  datatype: float
+  description: Vehicle idling
+
+MovementSensor:
+  type: sensor
+  datatype: float
+  description: Vehicle movement detected
+
+ECO.BrakingValue:
+  type: sensor
+  datatype: float
+  description: Braking value
+
+ECO.AccelerationValue:
+  type: sensor
+  datatype: float
+  description: Acceleration value
+
+ECO.CorneringValue:
+  type: sensor
+  datatype: float
+  description: Cornering value
+
+UnplugDetection:
+  type: sensor
+  datatype: float
+  description: device unplug detection
+
+CurrentLTEBand:
+  type: sensor
+  datatype: float
+  description: Current LTE band
```

Comment:
I'm unclear on the interpretation of any of these.

Reply:
I need to merge the ignition status for Astreon asap; we can leave the above out of the scope of the current pr. Thoughs?

Reply:
It's still in the PR. Am I supposed to ignore it? I don't understand.